static/src/javascripts/projects/common/modules/commercial/track-rubicon-ad-resize.js

```diff
@@ -0,0 +1,60 @@
+define([
+    'common/modules/commercial/track-ad'
+], function (
+    trackAd
+) {
+    function trackRubiconAdResize(id) {
+        return trackAd.track(id)
+            .then(getIframeId)
+            .then(listenForRubicon)
+            .then(getDims);
+
+        function getIframeId(isLoaded) {
+            if (!isLoaded) {
+                return false;
```

Comment:
Nope, you're right. I wouldn't be surprised if there was some optimisation done when parameters and return values remain consistent.